### ISSUE TITLE
JsonMapper bean already declared in Spring

### ISSUE THREAD

**Issue by PhilippS93** [camunda-community-hub/spring-zeebe#244]
Hello,

while using the version 8.0.11 or newer, I noticed the following error when starting Spring using spring cloud:

`The bean 'jsonMapper', defined in class path resource [org/springframework/cloud/function/context/config/ContextFunctionCatalogAutoConfiguration$JsonMapperConfiguration.class], could not be registered. A bean with that name has already been defined in class path resource [io/camunda/zeebe/spring/client/config/ZeebeClientStarterAutoConfiguration.class] and overriding is disabled.
`

The problem seems to be https://github.com/camunda-community-hub/spring-zeebe/commit/607084cf1c03da5ac642e8bd628fa2046b65cb68 because the bean is already declared.

Spring suggests to enable bean overriding, but this is disabled by default, because it is not a good practice and can lead so serious problems.

```
Action:

Consider renaming one of the beans or enabling overriding by setting spring.main.allow-bean-definition-overriding=true
```

Best,
Philipp

**Comment by berndruecker** [comment]
Hi @PhilippS93 - this should be fixed with https://github.com/camunda-community-hub/spring-zeebe/commit/bf5ff0c330139791e7c224a4ca1202fb58a09feb - could you confirm this probably with the current SNAPSHOT? I could build a 8.1.1 with this fix if helpful?

**Comment by PhilippS93** [comment]
Thanks @berndruecker ,

we are using 
`implementation group: 'io.camunda', name: 'spring-zeebe-starter', version: '8.1.0'`
currently. 
Do you mean `implementation group: 'io.camunda', name: 'spring-zeebe-starter', version: '8.1.0-SNAPSHOT'`?

**Comment by berndruecker** [comment]
Yes - exactly. If you confirm that this is fixed in `8.1.0-SNAPSHOT` I am happy to release a fix

**Comment by PhilippS93** [comment]
Alight, can you share the repository on which the snapshot version is available. Currently, we are using 
`mavenCentral` for the 8.1.0 version.

**Comment by berndruecker** [comment]
@PhilippS93  It is in our Artifactory:
https://artifacts.camunda.com/artifactory/zeebe-io-snapshots/io/camunda/spring-zeebe/8.1.1-SNAPSHOT/

**Comment by PhilippS93** [comment]
Alright, it seems to work 👍 !

**Comment by berndruecker** [comment]
Thanks! Released 8.1.1 - might take a bit until it finally arrives on Maven Central.
https://github.com/camunda-community-hub/spring-zeebe/releases/tag/8.1.1
